### ISSUE TITLE
Auto generated bios don’t include links to mentioned users or organizations

### ISSUE THREAD

**Issue by nobody5050** [benawad/dogehouse#1600]
**Is your feature request related to a problem? Please describe.**
When you create a new account at doghouse with GitHub, it imports your GitHub bio. It Doesn’t however link to any organizations or users you’ve mentioned, for instance @dahlia-os or @mcpi-devs in my case

**Describe the solution you'd like**
Make anything directly following an @ be a link to 'GitHub.com/[insert username here]'

**Describe alternatives you've considered**
An alternative is to leave it like it is, where mentioned users and organizations do not link.

**Additional context**

Here’s an example with my profile
![example with my profile](https://user-images.githubusercontent.com/32989720/112661008-10fe4c00-8e24-11eb-8e6f-c1cf5e79e10c.jpeg)

And here’s with a bot I found on a random room
![random bot](https://user-images.githubusercontent.com/32989720/112661635-bd403280-8e24-11eb-8b8c-b5f36b54aa85.png)

**Comment by f1shy-dev** [comment]
For this to work, we would need to add some form of WYSIWYG editor to the bio section so that we could add links/mentions. This isn't added yet, but adding it would also mean we would need to add it for non-github users.

**Comment by nobody5050** [comment]
> For this to work, we would need to add some form of WYSIWYG editor to the bio section so that we could add links/mentions. This isn't added yet, but adding it would also mean we would need to add it for non-github users.

That is true. Either that or something to acknowledge that the behavior is unexpected.
